site/assets/scss/_toc.scss

```diff
@@ -40,3 +40,42 @@
     }
   }
 }
+
+.bd-toc-toggle {
+  display: flex;
+  align-items: center;
+
+  @include media-breakpoint-down(sm) {
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  @include media-breakpoint-down(md) {
+    border: 1px solid $border-color;
+    @include border-radius(.4rem);
+
+    &:hover,
+    &:focus,
+    &:active,
+    &[aria-expanded="true"] {
+      color: var(--bd-violet);
+      background-color: $white;
+      border-color: var(--bd-violet);
+    }
+
+    &:focus,
+    &[aria-expanded="true"] {
+      box-shadow: 0 0 0 3px rgba(var(--bd-violet-rgb), .25);
+    }
+  }
+
+  @include media-breakpoint-up(md) {
+    pointer-events: none;
+  }
```

Comment:
I think the button should be hidden on large screens, and replaced by a heading or something: currently, when navigating with keyboard, I'm still able to activate the button, which makes the ToC flicks visually (and makes `aria-*` irrelevant).